Python_Module/MangDang/mini_pupper/capabilities.py

```diff
@@ -0,0 +1,19 @@
+class Capabilities:
+
+    def __init__(self, image_dir='/var/lib/mini_pupper_bsp'):
+        self.capabilities = {}
+        self.capabilities['version'] = 'mp2'
```

Comment:
```suggestion
        self.capabilities['version'] = 'mini_pupper_2'
```

It may be better to write here without omission.